app/code/Magento/Developer/Model/Logger/Handler/Debug.php

```diff
@@ -48,30 +53,34 @@ public function __construct(
     /**
      * @inheritdoc
      */
-    public function isHandling(array $record)
+    public function isHandling(array $record): bool
     {
         if ($this->deploymentConfig->isAvailable()) {
-            return
-                parent::isHandling($record)
-                && $this->isLoggingEnabled();
+            return parent::isHandling($record) && $this->isLoggingEnabled();
         }
 
         return parent::isHandling($record);
     }
 
     /**
-     * Check that logging functionality is enabled.
+     * Check that logging functionality is enabled
      *
      * @return bool
+     * @throws FileSystemException
+     * @throws RuntimeException
      */
     private function isLoggingEnabled(): bool
     {
-        $configValue = $this->deploymentConfig->get(ConfigOptionsList::CONFIG_PATH_DEBUG_LOGGING);
+        $configValue = $this->deploymentConfig->get(
+            ConfigOptionsList::CONFIG_PATH_DEBUG_LOGGING
+        );
+
         if ($configValue === null) {
             $isEnabled = $this->state->getMode() !== State::MODE_PRODUCTION;
         } else {
-            $isEnabled = (bool)$configValue;
+            $isEnabled = (bool) $configValue;
```

Comment:
```suggestion
            $isEnabled = (bool)$configValue;
```

Reply:
Hello @ihor-sviziev.
Thank you for your suggestion, but I'm not sure if this is a good idea.
Thank you again )))